hazelcast-client/src/test/java/com/hazelcast/client/listeners/AbstractListenersOnReconnectTest.java

```diff
@@ -393,14 +484,18 @@ public void run() throws Exception {
                 if (smartRouting) {
                     Collection<Member> members = clientInstanceImpl.getClientClusterService().getMemberList();
                     for (ClientEventRegistration registration : registrations) {
-                        Member registeredSubscriber = registration.getSubscriber();
+                        Connection registeredSubscriber = registration.getSubscriber();
+                        boolean contains = false;
+                        for (Member member : members) {
+                            contains |= registeredSubscriber.getEndPoint().equals(member.getAddress());
```

Comment:
There is not change in test behaviour. 
Since `registration.getSubscriber()` returning Connection instead of Member now, I had to change the verification code.